github/checks_test.go

```diff
@@ -460,23 +460,20 @@ func TestChecksService_CreateCheckSuite(t *testing.T) {
 	}
 }
 
-func TestChecksService_RequestCheckSuite(t *testing.T) {
+func TestChecksService_ReRequestCheckSuite(t *testing.T) {
 	client, mux, _, teardown := setup()
 	defer teardown()
 
-	mux.HandleFunc("/repos/o/r/check-suite-requests", func(w http.ResponseWriter, r *http.Request) {
+	mux.HandleFunc("/repos/o/r/check-suites/1/rerequest", func(w http.ResponseWriter, r *http.Request) {
 		testMethod(t, r, "POST")
 		testHeader(t, r, "Accept", mediaTypeCheckRunsPreview)
-		testBody(t, r, `{"head_sha":"deadbeef"}`+"\n")
+		w.WriteHeader(http.StatusCreated)
 	})
-	opt := RequestCheckSuiteOptions{
-		HeadSHA: "deadbeef",
-	}
-	resp, err := client.Checks.RequestCheckSuite(context.Background(), "o", "r", opt)
+	resp, err := client.Checks.ReRequestCheckSuite(context.Background(), "o", "r", 1)
 	if err != nil {
-		t.Errorf("Checks.RequestCheckSuite return error: %v", err)
+		t.Errorf("Checks.ReRequestCheckSuite return error: %v", err)
 	}
-	if resp.StatusCode != 200 {
-		t.Errorf("Checks.RequestCheckSuite return %+v, want 200", resp.StatusCode)
+	if resp.StatusCode != 201 {
```

Comment:
I know that you didn't write this code originally, but if you don't mind making it a bit more idiomatic Go while we are here, could you please change this to:

```go
if got, want := resp.StatusCode, http.StatusCreated; got != want {
	t.Errorf("Checks.ReRequestCheckSuite = %v, want %v", got, want)
}
```